mirrord-agent/src/file.rs

```diff
@@ -176,7 +180,55 @@ impl FileManager {
 
         debug!("FileManager::write -> Trying to close file {:#?}", file);
 
-        Ok(CloseFileResponse)
+        Ok(CloseFileResponse { result: 0 })
+    }
+
+    pub(crate) fn opendir(
```

Comment:
Do we need this distinction? Shouldn't `opendir` be the same as `open`? Have you checked how Rust implements this?